docs/specs/layers.md

```diff
@@ -71,3 +73,79 @@ decrease it ("stretching" the layer timeline).
     </script>
 </lottie-playground>
 
+<h4 id="precomposition-time-remap">Time Remap</h4>
+
+The `tm` property specifies a time remap function, which offers full control over the precomp
+timeline.  It maps the current layer time (in the frame index $[in..out]$ domain) to a precomp
+time expressed in seconds, and evaluates all animatable precomp properties based on the new
+time value:
+
+$$t\prime = TM(t) * FPS$$
```

Comment:
Is it allowed for the time remap to only cover a portion of the animation? If so, what's the behavior? I would guess frames outside of the remap would play as normal. It may be worth explicitly calling this out.

Reply:
The input/domain range is fixed ($[Layer.ip .. Layer.op]$), but the output/codomain range is arbitrary. Expanded the description.

Reply:
Usually there's no timing requirements for keyframes (other than being strictly increasing), I don't recall `tm` being special in that case. Values before the first keyframe or after the last just have the same value as that keyframe.

Reply:
Yeah - Brandon, think of `tm` as just a regular animatable property (which it basically is).  There are no special rules around its input range, it can be evaluated for the whole layer timeline just like all other animatable properties.